accelerator/apis.c

```diff
@@ -53,7 +53,7 @@ status_t api_get_tips(const iota_client_service_t* const service, char** json_re
   return ret;
 }
 
-status_t api_get_tips_pair(const iota_config_t* const tangle, const iota_client_service_t* const service,
+status_t api_get_tips_pair(const iota_config_t* const conf, const iota_client_service_t* const service,
```

Comment:
I don't think the renaming is related to this PR.
And actually I prefer `tangle` to `conf`, since `iota_config_t` is only about tangle,
and `conf` is more like all configurations in TA.  :smiley:

Reply:
how about `tangle_conf`?

Reply:
Alternatively, use `iconf` for naming the instance of `iota_config_t.

Reply:
OK